index.html

```diff
@@ -1,54 +1,269 @@
+              <ul>
+                <li> Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <article>A daily selection of privately personalized reads;
+              no accounts or sign-ups required.</article>
+          </div>
+
+          <section class="button">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+            <div class="proyect">
+              <a href="#"><button>See Proyect</button></a>
+            </div>
+          </section>
+
+        </div>
+      </section>
+
+      <section class="aboutmyself">
+        <h2>About Myself</h2>
+
+        <article class="offer">
+          <p>
+            Hello I’m a software developer! I can
+            help you build a product , feature or
+            website. Look through some of my
+            work and experience! If you like what
+            you see and have a project you need
+            coded, don’t hesitate to contact me.
+          </p>
+        </article>
+        <section class="contact">
+          <p>LET’S CONNECT</p>
+        </section>
+
+        <section class="socialmedia-botton">
+          <ul>
+            <li><img src="./images/git.png" alt=""></li>
+            <li><img src="./images/I.png" alt="">
+              <img src="./images/N.png" alt="">
+            <li><img src="./images/hand.png" alt=""></li>
+            <li class="twitter"><img src="./images/twitter.png" alt=""></li>
+            </li>
+            <li><img src="./images/m.png" alt=""></li>
+          </ul>
+        </section>
+
+        <div class="get">
+          <a href="#"><button>Get my resume</button></a>
+        </div>
+
+
+        <section class="lenguage">
+
+          <div class="legheader">
+            <p>
+              Languages
+            </p>
+            <img class="arrowdown" src="./images/arrow-down.png" alt="">
+          </div>
+
+          <div class="logo">
+            <div class="leng">
+              <div class="image"><img src="./images/Ellipse 1.png" alt=""></div>
+              <p>Javascript</p>
+            </div>
+              
+            <div class="leng">
+              <div class="image"><img src="./images/Ellipse 1-1.png" alt=""></div>
+              <p>HTML</p>
+            </div>
+
+            <div class="leng">
+              <div class="image"><img src="./images/Ellipse 1-2.png" alt=""></div>
+              <p>CSS</p>
+            </div>
+          </div>
+
+          <div class="frame">
+
+            <div class="work">
+              <p>Frameworks</p>
+              <div class="image"><img src="./images/arrow-right.png" alt=""></div>
+             
+            </div>
+              <hr>
+            <div class="skills">
+              <p>Skills</p>
+              <div class="image"><img src="./images/arrow-right.png" alt=""></div>
+            </div>
+            <hr>
+          </div>
+        </section>
```

Comment:
- [ ] Nice job designing your section according to figma, 👍 Please do a little more effort to adjust the **font-weight** of the heading according to figma, so that it will provide a better user experience. 😊 Please remember replicating exact Figma templates gives confidence to recruiters in your ability to replicate business requirements as regards design 😄

#### Figma design:
![f4](https://user-images.githubusercontent.com/105105262/229162813-c215e528-5e8d-4306-a98d-819b10176046.PNG)

#### Your design:
![y4](https://user-images.githubusercontent.com/105105262/229162853-191bc366-e81b-413c-a183-754aadfef112.PNG)

_Please make sure to adjust the font weights of the technologies in each category(e.g., "JavasScript", "HTML" and "CSS"), as I forgot to mark them. Apologies_ 😥

- [ ] Also, please wrap all of the technology categories ("Languages", "Frameworks" and "Skills")  in a `ul` tag, and each of them in a `li` tag. Similarly, please make sure that the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`, as it is very necessary for accessibility and semantic purposes. 👍